packages/config/config/devices/0x0312/mp21zd.json

```diff
@@ -0,0 +1,177 @@
+{
+	"manufacturer": "Minoston",
+	"manufacturerId": "0x0312",
+	"label": "MP21ZD",
+	"description": "Smart Plug Dimmer",
+	"devices": [
+		{
+			"productType": "0xff00",
+			"productId": "0xff0d"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "Status LED Configuration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED on when load off, LED off when load on",
+					"value": 0
+				},
+				{
+					"label": "LED on when load on, LED off when load off",
+					"value": 1
+				},
+				{
+					"label": "LED always off",
+					"value": 2
+				},
+				{
+					"label": "LED always on",
+					"value": 3
+				}
+			]
+		},
+		"4": {
+			"label": "Auto-Off Timer",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"6": {
+			"label": "Auto-On Timer",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"7": {
+			"label": "Night Light Brightness",
+			"valueSize": 1,
+			"unit": "10%",
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 2,
+			"unsigned": true
+		},
+		"8": {
+			"label": "State After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				},
+				{
+					"label": "Last state",
+					"value": 2
+				}
+			]
+		},
```

Comment:
We now have [templates](https://github.com/zwave-js/node-zwave-js/blob/6c4d9d026a44d656e3b4d1011d7b1215cfa29ce3/packages/config/config/devices/templates/master_template.json#L256-L405) for the variants of this config option. Please choose the matching one and use it.